### ISSUE TITLE
how to run game???????????

### ISSUE THREAD

**Issue by ethanlindley** [packetflooder/toontown_2003#1]
this game not working!!!!!!!!!????????????

**Comment by xseano** [comment]
stolen code

**Comment by xseano** [comment]
skids

**Comment by xseano** [comment]
stolen code * skids = this.shit

**Comment by xseano** [comment]
skiiiiiids

**Comment by xseano** [comment]
har ahoy! sTolEN CoDe

**Comment by packetflooder** [comment]
Yeah, code that i dumped from the 2003 client, is totally stolen. Am i right? And toontown legacy (2003) is not using tth localizers? Am i right on that too?

**Comment by Bob693** [comment]
Real professional guys

**Comment by Bob693** [comment]
![unknown](https://user-images.githubusercontent.com/31153210/33629663-7722b0aa-d9ca-11e7-8f77-e9736521f322.png)

**Comment by packetflooder** [comment]
https://cdn.discordapp.com/attachments/378254550907355136/387705349408423942/unknown.png Hmm...
https://cdn.discordapp.com/attachments/378254550907355136/387705544649211916/unknown.png

**Comment by packetflooder** [comment]
If you wanna call someone out for stealing, go look at your code first.

**Comment by packetflooder** [comment]
Is it really that hard to fix the dumped 2003 localizers?

**Comment by packetflooder** [comment]
@xseano  dmca me bud

**Comment by packetflooder** [comment]
@xseano @ethanlindley  No dmca public, you all are liars..

https://github.com/github/dmca It would be here, keep lying about a DMCA Takedown.
